src/systems/scene-preview-camera-system.js

```diff
@@ -14,10 +17,18 @@ export class ScenePreviewCameraSystem {
   }
 
   tick() {
+    viewingCamera = viewingCamera || document.getElementById("viewing-camera");
+    uiRoot = uiRoot || document.getElementById("ui-root");
+    const isGhost = uiRoot && uiRoot.firstChild && uiRoot.firstChild.classList.contains("isGhost");
+    const entered = viewingCamera && viewingCamera.sceneEl.is("entered");
     for (const el of this.entities) {
```

Comment:
This should probably be a regular for loop. `for of` allocates.

Reply:
fixed